packages/date-picker/src/_subcomponents/DateInput/DateInput.module.scss

```diff
@@ -1,3 +1,9 @@
+@import "~@kaizen/design-tokens/sass/spacing";
+
 .dateInput {
   width: 100%;
+
+  input {
+    margin-top: $spacing-xs;
+  }
```

Comment:
As opposed to selecting `input`, create a new class and attach it to the Input instead.

I dived in a little more and realised `TextField` (I thought it could be used in `DateInput`, but it seems the icon prevents it from doing so, so sadly can't be done) also has the same issue.
Could you possibly fix the spacing there too? 🙏🏻

Reply:
Thanks @HeartSquared, that has been updated now.